htmd/protocols/production_v6.py

```diff
@@ -173,16 +199,48 @@ def _findFiles(self, inputdir):
                                    'Please set the Production.acemd.{f:} property to '
                                    'point to the {f:} file'.format(f=field, i=inputdir))
 
-            if self.useconstraints and self.acemd.consref is None:
-                self.acemd.consref = self.acemd.coordinates
+            if self._version == 2:
+                if self.useconstraints and self.acemd.consref is None:
+                    self.acemd.consref = self.acemd.coordinates
 
     def _amberFixes(self):
         # AMBER specific fixes
         if self.acemd.structure.endswith('.prmtop'):
             self.acemd.parmfile = self.acemd.parameters
             self.acemd.parameters = None
-            self.acemd.scaling14 = 0.8333333
-            self.acemd.amber = 'on'
+            if self._version == 2:
+                self.acemd.scaling14 = '0.8333333'
+                self.acemd.amber = 'on'
+
+    def _fb_potential2restraints(self, inputdir):
+        from htmd.molecule.molecule import Molecule
+        restraints = list()
+
+        fb_box = np.array(self.fb_box)
+        # convert fb_box to width
+        width = list(np.concatenate(np.diff(np.array([fb_box[::2], fb_box[1::2]]), axis=0)))
+
+        # If fb_box is not symmetrical
+        if not np.all(fb_box[::2] == -fb_box[1::2]):
+            # convert fb_box and fb_reference to fbcentre and width
+            mol = Molecule(os.path.join(inputdir, self.acemd.structure))
+            mol.read(os.path.join(inputdir, self.acemd.coordinates))
+            fb_refcentre = np.concatenate(mol.coords[mol.atomselect(self.fb_reference)].mean(axis=0))
```

Comment:
`fb_refcentre = mol.get('coords', sel=self.fb_reference).mean(axis=0).squeeze()  `
should do the same but cleaner

Reply:
I'm always doing concatenate and forgetting about squeeze... :(

Reply:
flatten would also work but I prefer squeeze when it's an empty dimension